src/drawer/drawer.styles.tsx

```diff
@@ -0,0 +1,109 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import { ClickableIcon } from "../shared/clickable-icon";
+import { Text } from "../text";
+
+// =============================================================================
+// STYLE INTERFACE, transient props are denoted with $
+// See more https://styled-components.com/docs/api#transient-props
+// =============================================================================
+interface IStyleProps {
+    $show: boolean;
```

Comment:
the parent's `show` prop is an optional. Should make the typing be `boolean | undefined`. Alternatively can assign a default val to `show`